account_payment_sale/tests/common.py

```diff
@@ -1,10 +1,10 @@
 # Copyright 2018 Camptocamp
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import TransactionCase
 
 
-class CommonTestCase(SavepointCase):
+class CommonTestCase(TransactionCase):
```

Comment:
I would prefer to keep SavepointCase

Reply:
SavepointCase is deprecated from 15 and TransactionCase has been made as SavepointCase (yeah.... it sucks :sob:)